examples/simple-nft-minter/index.mjs

```diff
@@ -0,0 +1,72 @@
+import { Reach, test } from '@reach-sh/stdlib';
+const stdlib = new Reach(process.env, { REACH_NO_WARN: 'Y' });
+
+const startingBalance = stdlib.parseCurrency(100);
+const [minter, receiver] = await stdlib.newTestAccounts(2, startingBalance);
+minter.setDebugLabel('Minter');
+receiver.setDebugLabel('Receiver');
+
+const gasLimit = 5000000;
+if (stdlib.connector != 'ALGO') {
+    minter.setGasLimit(gasLimit) && receiver.setGasLimit(gasLimit)
+};
+
+const mintAddr = minter.getAddress();
+console.log(`Minter's address is ${mintAddr}`);
+const recAddr = receiver.getAddress();
+console.log(`Receiver's address is ${recAddr}`);
+
+const minterAddrFormat = await stdlib.formatAddress(minter);
+console.log(`The minter's formatted address is ${minterAddrFormat}`);
+const receiverAddrFormat = await stdlib.formatAddress(receiver);
+console.log(`The receiver's formatted address is ${receiverAddrFormat}`);
+
+const fmt = (x) => stdlib.formatCurrency(x, 4);
+const getBal = async (who, tok) => tok ? (await stdlib.balanceOf(who, tok)) : fmt(await stdlib.balanceOf(who));
+
+const logBalance = async (acc, tok) => {
```

Comment:
In these two functions, you have an "acceptable" path on `tok` being `undefined`. I think it is kinder to use `(who, tok = false)`, to better communicate to the reader that often you expect it to not be provided